js/src/mpl_widget.js

```diff
@@ -240,6 +240,13 @@ export class MPLCanvasModel extends widgets.DOMWidgetModel {
         }
     }
 
+    generateMimeBundle() {
+        return Promise.resolve({
+            'text/html': `<img src="${this.offscreen_canvas.toDataURL('image/jpeg', 1.0)}"
```

Comment:
why `jpeg`? FWIW the backend provides a PNG. The data comes from here:
https://github.com/matplotlib/matplotlib/blob/56d9e7cb1718b740dee717ac0a208f7eae9db204/lib/matplotlib/backends/backend_webagg_core.py#L239-L242

Reply:
IDK actually, it's the first idea that came to my mind. We can do PNG then :)